### PR TITLE
media-video/vokoscreen: Add package vokoscreen

### PR DESCRIPTION
**Pull request by qiankehan**
Signed-off-by: Han Han <hanhanzhiyeqianke@gmail.com>

### CODE REVIEW

**Comment by qiankehan** [comment]
ping

File: media-video/vokoscreen/vokoscreen-2.5.8_beta.ebuild [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+BDEPEND=">=dev-qt/linguist-tools-${MINQT}"
+DEPEND=">=dev-qt/qtcore-${MINQT}
+		media-libs/gst-plugins-bad
+		media-sound/alsa-utils
+		x11-libs/libX11
+		media-sound/lame
+		media-libs/libv4l
+		x11-misc/xdg-utils
+		sys-process/lsof
+		>=media-video/ffmpeg-1.1.0
```
Comment (steils) [inline media-video/vokoscreen/vokoscreen-2.5.8_beta.ebuild]: There are no such old versions in the tree, so there is no need to specify the version.

File: media-video/vokoscreen/vokoscreen-2.5.8_beta.ebuild [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		media-sound/pulseaudio
+		$BDEPEND"
```
Comment (steils) [inline media-video/vokoscreen/vokoscreen-2.5.8_beta.ebuild]: `$BDEPEND` here is definitely wrong...
Also please sort the dependencies alphabetically.
See devmanual and skel.ebuild for reference about `{B,R}DEPEND`.

**Comment by a17r** [comment]
ping

**Comment by pajot** [comment]
I'd also like to see this in main tree. How can I help move this along? Github says all checks have passed.

**Comment by ajakk** [comment]
> I'd also like to see this in main tree. How can I help move this along? Github says all checks have passed.

Given the maintainer has not responded to requests for changes in more than a year, we can't do much but close this for lack of interest. In general, moving forward the proxy-maint project doesn't add new leaf packages (i.e. packages that are not added as dependencies) and the right place for those is now the [GURU overlay](https://wiki.gentoo.org/wiki/Project:GURU). If you are interested in maintaining the package there, please feel free!

**Pull request closed without merging**